docs/user/install/initial_setup.rst

```diff
@@ -0,0 +1,65 @@
+.. _initial_setup:
+
+
+Initial Setup
```

Comment:
@jeepurs Is this change from 'Onboarding' fine by you?

Reply:
Yup!